modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/daemon/SnapshotEventsPublisherDaemon.scala

```diff
@@ -20,11 +21,16 @@ trait SnapshotEventsPublisherDaemon[F[_]] {
 object SnapshotEventsPublisherDaemon {
   def make[F[_]: Async: Supervisor, E: TypeTag: Encoder](
     gossip: Gossip[F],
-    consensusEvents: Stream[F, E]
+    consensusEvents: Stream[F, E],
+    consensusStorage: ConsensusStorage[F, E, _, _, _, _, _, _]
   ): SnapshotEventsPublisherDaemon[F] =
     new SnapshotEventsPublisherDaemon[F] {
       def spawn: Daemon[F] = Daemon.spawn {
-        consensusEvents
+        consensusEvents.map { e =>
+          println(e)
+          e
+        }
```

Comment:
It think you don't need to println here, even if you need probably you need to change to logger

Reply:
i believe it's a leftover and should be removed. for tests, you should match and expect instead of visually check what's printed in console :)